message_ix_models/testing.py

```diff
@@ -201,3 +222,120 @@ def bare_res(request, context: Context, solved: bool = False) -> message_ix.Scen
 
     log.info(f"Clone to '{name}/{new_name}'")
     return base.clone(scenario=new_name, keep_solution=solved)
+
+
+#: Items with names that match (partially or fully) these names are omitted by
+#: :func:`export_test_data`.
+EXPORT_OMIT = [
+    "aeei",
+    "cost_MESSAGE",
+    "demand_MESSAGE",
+    "demand",
+    "depr",
+    "esub",
+    "gdp_calibrate",
+    "grow",
+    "historical_gdp",
+    "kgdp",
+    "kpvs",
+    "lakl",
+    "land",
+    "lotol",
+    "mapping_macro_sector",
+    "MERtoPPP",
+    "prfconst",
+    "price_MESSAGE",
+    "sector",
+]
```

Comment:
Thanks for adding the option to add additional items to exclude. As for the hard-coded exclusions, I would request ref_activity,ref_extraction,ref_new_capacity,ref_relation to be added, as this is only very seldomly needed.

Reply:
Done!